packages/cli/package.json

```diff
@@ -1,103 +1,106 @@
+    "author": "Zowe",
+    "license": "EPL-2.0",
+    "keywords": [
+        "credential manager",
+        "secure store",
+        "password"
     ],
-    "modulePathIgnorePatterns": [
-      "__tests__/__snapshots__/"
-    ],
-    "testPathIgnorePatterns": [
-      "<rootDir>/__tests__/__results__"
-    ],
-    "transform": {
-      ".(ts)": "ts-jest"
+    "publishConfig": {
+        "registry": "https://zowe.jfrog.io/zowe/api/npm/npm-local-release/"
     },
-    "testRegex": "(/__tests__/.*\\.(test|spec))\\.(ts|js)$",
-    "moduleFileExtensions": [
-      "ts",
-      "js"
-    ],
-    "testEnvironment": "node",
-    "collectCoverageFrom": [
-      "src/**/*.ts",
-      "!**/__tests__/**",
-      "!**/index.ts",
-      "!**/main.ts"
+    "repository": "https://github.com/zowe/zowe-cli-secrets-for-kubernetes",
+    "files": [
+        "lib",
+        "scripts"
     ],
-    "collectCoverage": false,
-    "reporters": [
-      "default",
-      "jest-junit",
-      "jest-stare",
-      "jest-html-reporter"
-    ],
-    "coverageReporters": [
-      "json",
-      "lcov",
-      "text",
-      "cobertura"
-    ],
-    "coverageDirectory": "<rootDir>/__tests__/__results__/unit/coverage"
-  },
-  "jest-stare": {
-    "resultDir": "<rootDir>/__tests__/__results__/jest-stare"
-  },
-  "jest-junit": {
-    "outputDirectory": "<rootDir>/__tests__/__results__/unit/junit"
-  },
-  "jest-html-reporter": {
-    "pageTitle": "Kubernetes secrets Plugin for Zowe CLI Test Results",
-    "outputPath": "<rootDir>/__tests__/__results__/results.html",
-    "includeFailureMsg": true
-  }
+    "main": "lib/index.js",
+    "typings": "lib/index.d.ts",
+    "imperative": {
+        "configurationModule": "lib/imperative/configuration.js"
+    },
+    "scripts": {
+        "clean": "rimraf lib",
+        "checkTestsCompile": "echo \"Checking that test source compiles...\" && tsc --project __tests__/test-tsconfig.json --noEmit ",
+        "circularDependencyCheck": "madge -c lib",
+        "lint": "eslint \"src/**/*.ts\" \"**/__tests__/**/*.ts\"",
+        "lint:src": "eslint \"src/**/*.ts\" --ignore-pattern \"**/__tests__/**/*.ts\"",
+        "lint:tests": "eslint \"**/__tests__/**/*.ts\"",
+        "prebuild": "npm run clean && npm run lint && echo Using TypeScript && tsc --version",
+        "build": "tsc --pretty && npm run checkTestsCompile && npm run circularDependencyCheck",
+        "prepublishOnly": "npm run build && bash ./scripts/bundleKeytar.sh",
+        "watch": "tsc --pretty --watch",
+        "watch:test": "jest --watch",
+        "installPlugin": "npm install && npm run clean && npm run build && zowe plugins install .",
+        "postuninstall": "echo \"CLI Profiles that you have created will need to be recreated because their credentials are stored in the credential manager that has been uninstalled. Refer to the available options in the help text to create a new profile 'zowe profiles create zosmf-profile -h'.\" 1>&2",
```

Comment:
Should we remove the `postuninstall` script, since npm no longer supports uninstall lifecycle scripts?